projects/plugins/jetpack/modules/search/customberg/lib/_base-styles.scss

```diff
@@ -0,0 +1,29 @@
+@import '~@wordpress/base-styles/z-index';
+
+// Fill in missing z-layer values from a more recent @wordpress/base-styles version.
+// TODO: Remove once @wordpress/base-styles is updated.
```

Comment:
Are we waiting on a new release of base-styles?

Reply:
Upgrading `base-styles` will break other parts of Jetpack, so I decided against performing the upgrade purely for our sake.